src/gretel_trainer/relational/report/report_template.html

```diff
@@ -58,7 +58,7 @@ <h1>Gretel Relational<br />Synthetic Report</h1>
       <h2>Table Relationships</h2>
 
       <p>
-        The primary and foreign keys for each table in the synthesized database and their 
```

Comment:
Just a note: I confirmed that deleting these trailing whitespaces at the ends of lines does **not** cause words to be jammed together in the final output

Reply:
I will permit this.